docker-compose.yaml

```
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  redis:
+    image: redis:latest
+    ports:
+      - 6379:6379
+    healthcheck:
+      test: ["CMD", "redis-cli", "--raw", "incr", "ping"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  data-generator:
+    build:
+      context: .
+      dockerfile: docker/data-generator/Dockerfile
```

```suggestion
      context: docker/data-generator
      dockerfile: Dockerfile
```

context에 path를 적고 dockerfile에는 dockerfile 이름을 적는게 돜룰인 것 같아요~

---

돜룰 지켜야죠 ㅋㅋㅋ 반영했습니다!